webpack/loaders/manifest-loader.cjs

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+let fs = require('fs');
+
+module.exports = function(content, map, meta) {
+  const contents = fs.readFileSync(this.resourcePath).toString();
+  const lines = contents.split(/[\r\n]+/);
+
+  let fileIdx = 0;
+  let importStr = '';
+  let outputStr = 'export default {';
+
+  for (let name of lines) {
+    if (/^\s*$/.test(name))
+      continue;
+    name = name.replace(/\\/g, '/');
+    name = name.replace(/^\//, '');
+
+    // Use static imports instead of dynamic ones to put files in the bundle.
+    const fileVar = `file${fileIdx++}`;
```

Comment:
This one's minor, but since we're already regexing, we might as well have it cut out potential whitespace instead of checking below.
```suggestion
  const lines = contents.trim().split(/\s*[\r\n]+\s*/);

  let importStr = '';
  let outputStr = 'export default {';

  for (const fileIdx in lines) {
    // normalize filepaths between windows / unix
    const name = lines[fileIdx].replace(/\\/g, '/').replace(/^\//, '');

    // Use static imports instead of dynamic ones to put files in the bundle.
    const fileVar = `file${fileIdx}`;
```